lib/active_model/serializer/adapter.rb

```diff
@@ -69,7 +72,8 @@ def lookup(adapter)
         # @api private
         def find_by_name(adapter_name)
           adapter_name = adapter_name.to_s.classify.tr('API', 'Api')
-          ActiveModel::Serializer::Adapter.const_get(adapter_name.to_sym) or # rubocop:disable Style/AndOr
+          ActiveModel::Serializer::Adapter.const_get(adapter_name.to_sym) ||
+            ActiveModel::Serializer::Adapter.const_get(adapter_name.pluralize.to_sym) or # rubocop:disable Style/AndOr
```

Comment:
can we either obey rubocop, or change the rubocop setting? 
disabling static analysis warnings via comments (i think) looks tacky.